_components/header/guidance/variants/basic/implementation.md

```diff
@@ -0,0 +1 @@
+{% include init-note.html %}
```

Comment:
I think it would be best if we moved this to note to the general guidance at the top of the page. What do you think?
<img width="1074" alt="image" src="https://user-images.githubusercontent.com/93996430/213205779-3f234808-9b9f-4b84-ae1d-1389ec74d1ef.png">

It would also be great if we could remove the empty "Using the..." headers, but that might need to be done in another issue.

Reply:
That works for me! I originally set it under the `Using the...` section because it was universal across the three pages but I didn't consider that Header has different variants.

What do you think of the [Modal](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-uswds-init-note-modal-header/components/modal/) and [Banner](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-uswds-init-note-modal-header/components/banner/) pages? I think the note goes well where it's at, but could potentially be moved to `Usability guidance`

Reply:
I think implementation is the right spot for these. I contemplated moving it higher up on the page but I landed on thinking it was most appropriate where it was.